src/Chapter1.hs

```diff
@@ -489,10 +492,9 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
-
-
+lastDigit :: Int -> Int
+lastDigit n = n `mod` 10
```

Comment:
Your implementation is almost correct :ok: 
Unfortunately, it returns negative numbers on negative inputs because of how `mod` works. Sometimes corner cases can be tricky to spot and fix...

Reply:
Thank you for the feedback. 
I'm now using the `abs()` function to also evaluate negative numbers 😃